spark-3.3/clickhouse-spark-it/src/test/scala/org/apache/spark/sql/clickhouse/single/ClickHouseDataTypeSuite.scala

```diff
@@ -176,7 +183,7 @@ class ClickHouseDataTypeSuite extends SparkClickHouseSingleTest {
   private def testDataType(valueColDef: String)(prepare: (String, String) => Unit)(validate: DataFrame => Unit)
     : Unit = {
     val db = "test_kv_db"
-    val tbl = "test_kv_tbl"
+    val tbl = s"test_kv_tbl_${valueColDef}".replace("(", "_").replace(")", "_").replace(",", "_")
```

Comment:
seems we don't need this change after disabling parallel testing

Reply:
This actually happens when testing against the docker I will disable also parallel testing and test

Reply:
So, I have run some tests related to old ClickHouse versions. Once I added a sleep there, it worked. I will try to understand why it happens only on those versions of ClickHouse. Maybe for now a conditional sleep will do the trick